to_do_list/src/Components/AssignmentsList.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Assignment from './Assignment';
+import '../Style/AssignmentsList.css';
+
+function AssignmentsList(props) {
+    return (
+        <ul className="assignmentsList">
+            {Object.entries(props.data).map((item) => {
+                return (
+                    <li key={item[0]}>
+                        <Assignment id={item[0]} isDone={item[1]['done']} changeItem={props.change} delete={props.delete}>
+                            {item[1]['assignment']}
```

Comment:
why this isn't one of the props for the component?
seems pretty self explanatory for me to get the full `item` (assignment) as a prop for the component that should display this data nicely.